docs/how_to/perf.md

```diff
@@ -243,7 +244,7 @@ The `mode` parameter can be set to
 * `symbolic` to only include symbolic operations
 * `all` to include all operations
 
-After the program finishes, navigate to chrome://tracing in a Chrome browser and load the `.json` file output by the profiler to inspect the results.
+After the program finishes, navigate to your browser's tracing (Example - chrome://tracing in a Chrome browser) and load the `profile_output.json` file, output by the profiler, to inspect the results.
```

Comment:
I thought the previous description was pretty good - the two commas you added are probably not needed.